app/scripts/modules/core/src/search/searchResult/SearchResultTabs.tsx

```diff
@@ -1,36 +1,32 @@
 import * as React from 'react';
-import { BindAll } from 'lodash-decorators';
+import { UISref } from '@uirouter/react';
+import { UIRouterContext } from '@uirouter/react-hybrid';
 
-import { ISearchResultData } from 'core/search/searchResult/SearchResults';
 import { ISearchResultType } from './searchResultsType.registry';
+import { ISearchResultSet } from '../infrastructure/infrastructureSearch.service';
 
 export interface ISearchResultTabsProps {
-  searchResultData: ISearchResultData[]
+  resultSets: ISearchResultSet[]
   activeSearchResultType: ISearchResultType;
-  onClick?: (group: ISearchResultType) => void;
 }
 
-@BindAll()
+@UIRouterContext
 export class SearchResultTabs extends React.Component<ISearchResultTabsProps> {
-  private handleClick(type: ISearchResultType) {
-    this.props.onClick && this.props.onClick(type);
-  }
-
   public render(): React.ReactElement<SearchResultTabs> {
-    const { searchResultData, activeSearchResultType } = this.props;
+    const { activeSearchResultType } = this.props;
+    const resultSets = this.props.resultSets.slice().sort((a, b) => a.type.order - b.type.order);
 
     return (
       <div className="search-groups">
```

Comment:
ul li may be.

Reply:
done.  This was a trivial change and definitely didn't require modifying 20 files.